### PR TITLE
Updating documentation on COMMIT_TAG build arg

### PR DESCRIPTION
**Pull request by merlin-northern**
Description of the build arg

* follow up from https://github.com/mendersoftware/mendertesting/pull/48

ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: .gitlab-ci-check-docker-build.yml [hunk 1]
```diff
@@ -24,6 +24,11 @@
 #   REGISTRY_MENDER_IO_USERNAME: Username for registry.mender.io
 #   REGISTRY_MENDER_IO_PASSWORD: Password for registry.mender.io
 #
+# Passes the following variables to the build:
+#   * COMMIT_TAG
```
Comment (lluiscampos) [inline .gitlab-ci-check-docker-build.yml]: `GIT_COMMIT_TAG`

Comment (merlin-northern) [reply .gitlab-ci-check-docker-build.yml]: [fixed](https://github.com/mendersoftware/mendertesting/pull/49/commits/b8304fb2b5a9a4e901f88b17f276ffbd4d8fb975#diff-8cc733627168077a8a8f6e5c95979adeR28)!

**Comment by merlin-northern** [comment]
> Thanks for the documentation ❤️
> 
> See comment below, though

love the heart!
thanks :)

**Comment by lluiscampos** [comment]
Running the Pipeline

**Pull request merged**